### ISSUE TITLE
Program 'spicetify.exe' failed to run

### ISSUE THREAD

**Issue by HypedDomi** [khanhas/spicetify-cli#1245]
## ℹ Computer information

- Spotify version: 1.1.71.560.gc21c3367
- Spicetify version: 2.7.2

## 📝 Provide detailed reproduction steps (if any)

1. Try any spicetify command

### ✔️ Expected result

Spicetify should work like normal and execute the command

### ❌ Actual result

Spicetify gets blocked by Windows with following message:
```
Program 'spicetify.exe' failed to run: Operation did not complete successfully because the file contains a virus or
potentially unwanted softwareAt line:1 char:1
+ spicetify backup apply
+ ~~~~~~~~~~~~~~~~~~~~~~.
At line:1 char:1
+ spicetify backup apply
+ ~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ResourceUnavailable: (:) [], ApplicationFailedException
    + FullyQualifiedErrorId : NativeCommandFailed
```
## 📷 Screenshots

![dDhowr](https://user-images.githubusercontent.com/50876016/141002824-3fa987d5-6e23-41bc-84a2-2d5f7b5a0270.png)

**Comment by TheGeeKing** [comment]
Duplicate of #1242 and Duplicate of #1244

**Comment by afonsojramos** [comment]
Still happening on 2.7.3? Don't have access to a Windows machine atm.

**Comment by TheGeeKing** [comment]
> Still happening on 2.7.3? Don't have access to a Windows machine atm.

Take exe and check it with virustotal

**Comment by HypedDomi** [comment]
Yeah works again. I close this

**Comment by afonsojramos** [comment]
Awesomeeee! I also had this problem with [Discrakt](https://github.com/afonsojramos/discrakt), thanks @TheGeeKing for the submission
